yarn.lock

```diff
@@ -18618,31 +18564,6 @@ node-sass@^4.13.1:
     stdout-stream "^1.4.0"
     "true-case-path" "^1.0.2"
 
-node-sass@^4.9.3:
```

Comment:
this is the duplicate `yarn.lock` that causes issues if you try to `yarn install` on `node@12.x`